core/frontend/src/components/vehiclesetup/overview/OnboardSensors.vue

```diff
@@ -144,6 +156,27 @@ export default Vue.extend({
         .filter((param) => param.value !== 0)
         .map((parameter) => decode(parameter.name, parameter.value))
     },
+    // DEV_ID params do not exist yet for temperature sensors, so here we detect the incoming message instead
```

Comment:
Agreed that longer term this should be fixed in firmware, and we should maybe leave a TODO in the code here with a URL to an issue in the ArduPilot repo, but this does seem like a useful workaround in the meantime.

Reply:
So.. why not fixing it in the firmware instead ? Is that hard ?

Reply:
Yes, not "hard" per say, but will make much more time to get in than the 30min I spent on this PR.

Plus it will only work from versions of ardupilot that are currently dev and upwards. This is a thing confusing users right now ("the sensor doesnt show in the table, that must mean it is not working")

Reply:
Can we have an issue to link though this issue in the ArduPilot repository or a draft a PR ?

Reply:
https://github.com/ArduPilot/ardupilot/issues/26895